libvast/src/uuid.cpp

```diff
@@ -109,4 +111,22 @@ bool operator<(const uuid& x, const uuid& y) {
   return std::lexicographical_compare(x.begin(), x.end(), y.begin(), y.end());
 }
 
+caf::expected<flatbuffers::Offset<fbs::UUID>>
+pack(flatbuffers::FlatBufferBuilder& builder, const uuid& x) {
+  auto data = builder.CreateVector(
+    reinterpret_cast<const uint8_t*>(&*x.begin()), x.size());
+  fbs::UUIDBuilder uuid_builder{builder};
+  uuid_builder.add_data(data);
+  return uuid_builder.Finish();
+}
+
+caf::error unpack(const fbs::UUID& x, uuid& y) {
+  if (x.data()->size() != uuid::num_bytes)
+    return make_error(ec::format_error, "wrong uuid format");
+  span<const byte, uuid::num_bytes> bytes{
+    reinterpret_cast<const byte*>(x.data()->data()), x.data()->size()};
+  y = uuid{bytes};
+  return ec::no_error;
```

Comment:
```suggestion
  return caf::none;
```

We usually use that, but I like `ec::no_error` better. Maybe we can stick to one style and then do a global sweep in one refactoring commit?

Reply:
Sounds good.